src/AdornedRulerPanel.cpp

```diff
@@ -1413,44 +1413,16 @@ void AdornedRulerPanel::ReCreateButtons()
    auto size = theTheme.ImageSize( bmpRecoloredUpSmall );
    size.y = std::min(size.y, GetRulerHeight(false));
 
-   auto buttonMaker = [&]
-   (wxWindowID id, teBmps bitmap, bool toggle)
-   {
-      const auto button =
-      ToolBar::MakeButton(
-         this,
-         bmpRecoloredUpSmall, bmpRecoloredDownSmall, 
-         bmpRecoloredUpHiliteSmall, bmpRecoloredHiliteSmall, 
-         bitmap, bitmap, bitmap,
-         id, position, toggle, size
-      );
-
-      position.x += size.GetWidth();
-      mButtons[iButton++] = button;
-      return button;
-   };
-   auto button = buttonMaker(OnTogglePinnedStateID, bmpPlayPointerPinned, true);
-   ToolBar::MakeAlternateImages(
-	   *button, 3,
-	   bmpRecoloredUpSmall, bmpRecoloredDownSmall,
-	   bmpRecoloredUpHiliteSmall, bmpRecoloredHiliteSmall,
-	   //bmpUnpinnedPlayHead, bmpUnpinnedPlayHead, bmpUnpinnedPlayHead,
-	   bmpRecordPointer, bmpRecordPointer, bmpRecordPointer,
-	   size);
-   ToolBar::MakeAlternateImages(
-	   *button, 2,
-	   bmpRecoloredUpSmall, bmpRecoloredDownSmall,
-	   bmpRecoloredUpHiliteSmall, bmpRecoloredHiliteSmall,
-	   //bmpUnpinnedPlayHead, bmpUnpinnedPlayHead, bmpUnpinnedPlayHead,
-	   bmpRecordPointerPinned, bmpRecordPointerPinned, bmpRecordPointerPinned,
-	   size);
-   ToolBar::MakeAlternateImages(
-      *button, 1,
+   const auto button = ToolBar::MakeButton(
+      this,
       bmpRecoloredUpSmall, bmpRecoloredDownSmall, 
       bmpRecoloredUpHiliteSmall, bmpRecoloredHiliteSmall, 
-      //bmpUnpinnedPlayHead, bmpUnpinnedPlayHead, bmpUnpinnedPlayHead,
       bmpPlayPointer, bmpPlayPointer, bmpPlayPointer,
-      size);
+      OnTogglePinnedStateID, position, true, size
+   );
+
+   position.x += size.GetWidth();
+   mButtons[iButton++] = button;
```

Comment:
Commit #2 eliminates the pin images.

Great!  You know I implemented the needed drawing code for the pinned play years ago, but -- the silly pin button image was NOT my idea and I never liked it.  Let it just be a triangle.

However -- what you also lose here, is the alternation of the color of the triangle between green for play, and red for record.

Are you sure you want that too?